default-views/auth/change-password.hbs

```diff
@@ -5,61 +5,35 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Change Password</title>
   <link rel="stylesheet" href="/common/css/bootstrap.min.css">
+  <link rel="stylesheet" href="/common/css/server.css">
 </head>
 <body>
 <div class="container">
-  <h4>Change Password</h4>
-</div>
-<div class="container">
-  <form method="post" action="/account/password/change">
-    {{#if error}}
+
+  <div class="page-header">
+    <h1>Change Password</h1>
+  </div>
+
+  {{#if validToken}}
+    <form method="post" action="/account/password/change">
+      {{> shared/error}}
       <div class="form-group">
-        <div class="row">
-          <div class="col-md-12">
-            <p class="text-danger"><strong>{{error}}</strong></p>
-          </div>
-        </div>
+        <label class="control-label" for="newPassword">New Password:</label>
+        <input type="password" class="form-control" name="newPassword"/>
       </div>
-    {{/if}}
 
-    {{#if validToken}}
-      <div class="form-group">
-        <div class="row">
-          <div class="col-md-12">
-            <label for="newPassword">New Password:</label>
+      <button type="submit" class="btn btn-primary">Change Password</button>
```

Comment:
Not directly related, but more of a note to self: We've added repeat password on another page, we probably should do that here as well.

But for this case, we need to find a good markup of the password component. I've used two divs with `col-md-3`in the previous design, so might revert something a bit here, unless you have a better suggestion?